tests/rollen/RolleAnlegen.spec.ts

```diff
@@ -0,0 +1,185 @@
+          return rolleCreationSuccessPage;
+        });
+        await test.step('Erfolgsseite prüfen', async () => {
+          await rolleCreationSuccessPage.checkSuccessPage(params);
+        });
+      });
+
+      test('Rolle anlegen und Gesamtübersicht prüfen', async () => {
+        const params: RolleCreationParams = { ...baseRolleParams, name: generateRolleName() };
+        const rolleCreationSuccessPage: RolleCreationSuccessPage = await test.step('Rolle anlegen', async () => {
+          const rolleCreationSuccessPage: RolleCreationSuccessPage = await rolleCreationPage.createRolle(params);
+          return rolleCreationSuccessPage;
+        });
+        const rolleDetailsView: RolleDetailsViewPage = await test.step('Zur Gesamtübersicht navigieren', async () => {
+          await rolleCreationSuccessPage.waitForPageLoad();
+          const rolleManagementViewPage: RolleManagementViewPage = await rolleCreationSuccessPage.backToResultList();
+          await rolleManagementViewPage.setPageSize('300');
+          return await rolleManagementViewPage.openGesamtuebersicht(params.name);
+        });
+
+        await test.step('Gesamtübersicht prüfen', async () => {
+          await rolleDetailsView.checkGesamtuebersicht(params);
+        });
+      });
+
+      // SPSH-2950
+      test('Rolle anlegen und Ergebnisliste prüfen', async () => {
+        const params: RolleCreationParams = { ...baseRolleParams, name: generateRolleName() };
+        const rolleCreationSuccessPage: RolleCreationSuccessPage = await rolleCreationPage.createRolle(params);
+        await rolleCreationSuccessPage.waitForPageLoad();
+        const rolleManagementViewPage: RolleManagementViewPage = await rolleCreationSuccessPage.backToResultList();
+        await rolleManagementViewPage.setPageSize('300');
+        await rolleManagementViewPage.checkIfRolleExists(params.name);
+        await rolleManagementViewPage.checkIfRolleHasServiceProviders(params.name, params.serviceProviders);
+      });
+
+      test('Als Nutzer mit einer neu angelegten Rolle anmelden', async ({ page }: PlaywrightTestArgs) => {
+        const params: RolleCreationParams = { ...baseRolleParams, name: generateRolleName() };
+        await rolleCreationPage.createRolle(params);
+        const organisationId: string = await getOrganisationId(page, testschuleName);
+        const rolleId: string = await getRolleId(page, params.name);
+        const user: UserInfo = await createPerson(
+          page,
+          organisationId,
+          rolleId,
+          generateNachname(),
+          generateVorname(),
+          undefined,
+          params.rollenart === rollenArtLabel.LERN ? await getOrganisationId(page, klasse1Testschule) : undefined,
+          new Set(
+            params.merkmale.includes(rollenMerkmalLabel.BEFRISTUNG_PFLICHT) ? [RollenMerkmal.BefristungPflicht] : []
+          )
+        );
```

Comment:
I think its a bit hard to read this method, can we maybe pull some stuff out for readability so it becomes: 
```suggestion
const user: UserInfo = await createPerson(
  page,
  organisationId,
  rolleId,
  nachname,
  vorname,
  undefined,
  klasseOrganisationId,
  merkmaleSet,
);
```